app/src/sharedTest/java/org/oppia/android/app/settings/profile/ProfileEditActivityTest.kt

```diff
@@ -141,6 +152,21 @@ class ProfileEditActivityTest {
     ApplicationProvider.getApplicationContext<TestApplication>().inject(this)
   }
 
+  @Test
+  fun testProfileEditActivity_hasCorrectActivityLabel() {
+    launch(
+      ProfileEditActivity::class.java
+    ).use { scenario ->
+      scenario.onActivity { activity ->
+        assertThat(activity.title).isEqualTo(
```

Comment:
Please add this comment too above this:
https://github.com/oppia/oppia-android/blob/55d0669b410ae69759ca0e07f4c08468a067e807/app/src/sharedTest/java/org/oppia/android/app/options/ReadingTextSizeActivityTest.kt#L117

Reply:
Done

Reply:
It has been placed at incorrect position and also you have mentioned only first line.

The comment is of two lines and it should be place above `assertThat` statement

Reply:
@shankarpriyank This is still pending.